examples/plot_9_plotly.py

```diff
@@ -0,0 +1,64 @@
+"""
+========================================
+Example with the plotly graphing library
+========================================
+
+sphinx-gallery supports examples made with the
+`plotly library <https://plotly.com/python/>`_. sphinx-gallery captures the
+``_repr_html_`` of plotly figure objects. Therefore you should return a figure
+object to display the figure.
```

Comment:
'sphinx-gallery is able to capture the ``_repr_html_`` of plotly figure objects (see :ref:`capture_repr`). To display your figure you should return the plotly figure object as the last line in your code block.' 

(not sure about my wording either)

Reply:
I am not sure 'return' is the right word here but not sure what would be better. 'The last line in your code block should be the plotly figure object' ?